libraries-data-io/pom.xml

```diff
@@ -55,6 +55,52 @@
             <artifactId>docx4j</artifactId>
             <version>${docx4j.version}</version>
         </dependency>
+	<!-- gson api -->
```

Comment:
The dependencies you added all are using tabs to indent.

Reply:
My eclipse editor wasn't showing any differences before. I am able to see the changes after changing the preferences for tab spaces. I fixed it now. Please check and let me know.